scripts/isolate_tests.py

```diff
@@ -59,6 +59,7 @@ def extract_docs_cases(path):
 def write_cases(f, tests):
     cleaned_filename = f.replace(".","_").replace("-","_").replace(" ","_").lower()
     for test in tests:
+        remainder = re.sub(r'^ {4}', '', test, 0, re.MULTILINE)
```

Comment:
@chriseth In reference to comments in other PR, I stripped this back to just the line I added and removed all auto linting.

Reply:
What exactly is this doing?

Reply:
I wrote this some time ago 😅.

Err. All the code examples when extracted are indented by 8 spaces, which violates the style guide, so before checking you need to remove 4 from each line.

Reply:
Please add that as a comment to the file.